app/templates/gulpfile.babel.js

```diff
@@ -0,0 +1,326 @@
+    .pipe(plugins.coffee, {bare: true})<% } %>
+    .pipe(plugins.sourcemaps.write, '.');<% } %>
+
+/********************
+ * Tasks
+ ********************/
+
+gulp.task('inject', cb => {
+    runSequence(['inject:js', 'inject:css'], cb);
+});
+
+gulp.task('inject:js', () => {
+    return gulp.src(paths.views.main)
+        .pipe(plugins.inject(gulp.src(_.union(
+            paths.client.scripts,
+            ['!client/**/*.spec.<%= scriptExt %>']
+        ), {read: false}), {
+            starttag: '<!-- injector:js -->',
+            endtag: '<!-- endinjector -->',
+            transform: (filepath) => '<script src="' + filepath.replace('/client/', '') + '"></script>'
+        }))
+        .pipe(gulp.dest('client'));
+});
+
+gulp.task('inject:css', () => {
+    return gulp.src(paths.views.main)
+        .pipe(plugins.inject(gulp.src('/client/**/*.css', {read: false}), {
+            starttag: '<!-- injector:css -->',
+            endtag: '<!-- endinjector -->',
+            transform: (filepath) => '<link rel="stylesheet" href="' + filepath.replace('/client/', '').replace('/.tmp/', '') + '">'
+        }))
+        .pipe(gulp.dest('client'));
+});
+
+// TODO: other styles
+gulp.task('inject:sass', () => {
+    return gulp.src('client/app/app.scss')
+        .pipe(plugins.inject(gulp.src(_.union(paths.client.styles, ['!' + paths.client.mainStyle]), {read: false}), {
+            starttag: '// injector',
+            endtag: '// endinjector',
+            transform: (filepath) => '@import \'' + filepath.replace('/client/app/', '').replace('/client/components/', '../components/') + '\';'
+        }))
+        .pipe(gulp.dest('client/app'));
+});
+
+gulp.task('styles', styles);<% if(filters.babel || filters.coffee) { %>
+
+gulp.task('transpile', () => {
+    return gulp.src(paths.client.scripts)
+        .pipe(transpile())
+        .pipe(gulp.dest('.tmp'));
+});<% } %>
+
+gulp.task('lint:scripts', cb => runSequence(['lint:scripts:client', 'lint:scripts:server'], cb));
+
+gulp.task('lint:scripts:client', () => gulp.src(paths.client.scripts).pipe(lintClientScripts()));
+
+gulp.task('lint:scripts:server', () => gulp.src(paths.server.scripts).pipe(lintServerScripts()));
+
+gulp.task('clean:tmp', () => gulp.src('.tmp', {read: false}).pipe(plugins.clean()));
+
+gulp.task('start:client', [<% if(filters.coffee) { %>'coffee', <% } %>'styles'], cb => {
+    whenServerReady(() => {
+        open('http://localhost:' + config.port);
+        cb();
+    });
+});
+
+gulp.task('start:server', () => {
+    process.env.NODE_ENV = process.env.NODE_ENV || 'development';
+    config = require('./server/config/environment');
+    nodemon('-w lib server/app.js')
+        .on('log', onServerLog);
+});
+
+gulp.task('watch', () => {
+    var testFiles = _.union(paths.client.test, paths.server.test);
+
+    plugins.livereload.listen();
+
+    plugins.watch(paths.client.styles)
+        .pipe(plugins.plumber())
+        .pipe(styles())
+        .pipe(plugins.livereload());
```

Comment:
For style reload use it as this 
```
  plugins.watch(paths.client.styles, function() {
    styles();
  });

  plugins.watch('.tmp/app/app.css', function() {
    plugins.livereload.changed('.tmp/app/app.css');
  });
```
It will inject css rather than reloading the page.
and Thanks for creating this gulpfile :smile: